app/src/main/java/com/iet/ietians_diary/RoadmapFragment.java

```diff
@@ -92,8 +93,8 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         RoadmapRecyclerViewAdapter RoadmapAdapter = new RoadmapRecyclerViewAdapter(list, getContext());
         roadmap.setAdapter(RoadmapAdapter);
 
-        StaggeredGridLayoutManager staggeredGridLayoutManager = new StaggeredGridLayoutManager(2, StaggeredGridLayoutManager.VERTICAL);
-        roadmap.setLayoutManager(staggeredGridLayoutManager);
+//        StaggeredGridLayoutManager staggeredGridLayoutManager = new StaggeredGridLayoutManager(2, StaggeredGridLayoutManager.VERTICAL);
```

Comment:
Please remove this comment, no need.

Reply:
Thanks for the review, I have removed the comment.